openmc/surface.py

```diff
@@ -2090,23 +2092,26 @@ def get_hexagonal_prism(edge_length=1., orientation='y',
     """
 
     l = edge_length
+    x = origin[0]
+    y = origin[1]
 
     if orientation == 'y':
-        right = XPlane(x0=sqrt(3.)/2*l, boundary_type=boundary_type)
-        left = XPlane(x0=-sqrt(3.)/2*l, boundary_type=boundary_type)
+        right = XPlane(x0=x + sqrt(3.)/2*l, boundary_type=boundary_type)
+        left  = XPlane(x0=x - sqrt(3.)/2*l, boundary_type=boundary_type)
```

Comment:
PEP8: Always use single space around assignment operator (=), i.e. should be `left = XPlane...`